aperturedb/ParallelLoader.py

```diff
@@ -14,15 +14,14 @@ class ParallelLoader:
 
     def __init__(self, db, dry_run=False):
 
-        self.db = db
+        self.db = db.create_new_connection()
```

Comment:
Is there something that the master will do with a new connection ? The worker will create a new connection in any case, right?
Perhaps I could use a primer on our connection related changes :)

Reply:
I simply create a new connection to test, at construction time, that the connection will work. Otherwise, it will fail and throw.